src/main/java/com/ibm/watson/etcd/EtcdUtilsFactory.java

```diff
@@ -154,20 +154,19 @@ protected ByteString pathToKey(String path, boolean ensureTrailingSlash) {
             path += "/";
         }
         ByteString bs = ByteString.copyFromUtf8(path);
-        return rootPrefix != null? rootPrefix.concat(bs) : bs;
+        return rootPrefix != null ? rootPrefix.concat(bs) : bs;
     }
 
     @Override
     public String getKvStoreType() {
         return ETCD_TYPE;
     }
 
-    private static final ByteString TEST_KEY =
-            ByteString.copyFromUtf8("__DUMMY_KEY_FOR_TESTING_CONNECTION");
+    private static final String TEST_PATH = "/__DUMMY_KEY_FOR_TESTING_CONNECTION";
 
     @Override
     public ListenableFuture<Boolean> verifyKvStoreConnection() {
-        return Futures.catching(Futures.transform(client.getKvClient().get(TEST_KEY).countOnly()
+        return Futures.catching(Futures.transform(client.getKvClient().get(pathToKey(TEST_PATH, false)).countOnly()
```

Comment:
is the `.countOnly()` what makes it cool to `.get()` a key that doesn't exist without throwing an error?

Reply:
I think even without that it's cool, it will just return an empty result (the "get" operation is actually generic and takes either a range or single key, as well as other filtering criteria, so it can return zero or more results).

Reply:
that's tight